modules/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/introspection/EmbeddedDocExtractor.java

```diff
@@ -56,6 +58,12 @@
      */
     protected static final String README = "ReadMe.md";
 
+    protected static List<Extension> MD_EXTENSIONS = List.of(TablesExtension.create());
+
+    protected static Parser MD_PARSER = Parser.builder().extensions(MD_EXTENSIONS).build();
```

Comment:
This is safe only if everything is single-threaded (which I think it is actually so it'd be ok).
Otherwise you could just have statics for a `Parser.Builder` and a `HtmlRenderer.Builder` and build a new one each time.

Reply:
I indeed hesitated, and i think they can be shared